examples/vector_fe/vector_fe_ex6/solution_function.h

```diff
@@ -0,0 +1,179 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2023 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+// libMesh includes
+#include "libmesh/function_base.h"
+
+// Example includes
+#include "div_grad_exact_solution.h"
+
+// C++ includes
+#include <memory>
+
+using namespace libMesh;
+
+#ifndef SOLUTION_FUNCTION_H
+#define SOLUTION_FUNCTION_H
```

Comment:
Minor comment but we conventionally place the include guards as the first non-comment lines of code in each header, right after the license info.

Reply:
Thanks. I can definitely do that, but then this would be the only vector fe example following that convention.
I would prefer to either change all of them or none?

Reply:
Yeah, after commenting I realized that this was probably just copy/pasted from elsewhere. I would say leave it as-is and then if anyone gets around to it, make a separate PR fixing it for all the vector FE examples.

Reply:
Done. #3639 should make this consistent as well. Feel free to resolve the conversation if you're happy. :)